README.md

```diff
@@ -1,16 +1,21 @@
 # Ideabox
 
 ### Summary
-
+An interactive repository of wit and wisdom-- all customizable by the user!
 ### How it Works
-- 
+- Open "index.html" on the Terminal.
```

Comment:
There is a link for the deployed page in "Check it out" so if we are looking from a user view, they will click check it out rather than use the terminal to demo unless they are cloning and developing on the project. Otherwise, great job!